src/functionaltests/java/com/ericsson/ei/subscriptions/content/SubscriptionContentSteps.java

```diff
@@ -0,0 +1,158 @@
+package com.ericsson.ei.subscriptions.content;
+
+import com.ericsson.ei.controller.model.Subscription;
+import com.ericsson.ei.controller.model.SubscriptionResponse;
+import com.ericsson.ei.utils.FunctionalTestBase;
+import com.ericsson.ei.utils.HttpRequests;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Ignore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import javax.annotation.PostConstruct;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+
+
+@Ignore
+public class SubscriptionContentSteps extends FunctionalTestBase {
+
+    @LocalServerPort
+    int applicationPort;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SubscriptionContentSteps.class);
+    private HttpRequests request;
+    private ResponseEntity response;
+
+    private String validSubscriptionFile = "src/functionaltests/resources/ValidSubscription.json";
```

Comment:
private static final and variable name should be like VALID_SUBSCRIPTION_FILE - java code convention